app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, source_file, destination_path = command.split()
+    if "/" in destination_path:
+        os.makedirs(os.path.dirname(destination_path), exist_ok=True)
+    with (open(source_file, "r") as copied_file,
```

Comment:
There is a function in `shutil` library that does all this in one line. Also how does this code handle cases when input file does not exist?